controllers/operator/manager.go

```diff
@@ -120,6 +120,27 @@ func (m *ODLMOperator) ListOperandRequests(ctx context.Context, label map[string
 	return requestList, nil
 }
 
+// ListOperandRequestsForRegistry list all the OperandRequests
+// using the specific operandregistry
+func (m *ODLMOperator) ListOperandRequestsForRegistry(ctx context.Context, registry types.NamespacedName) (requestList []apiv1alpha1.OperandRequest, err error) {
```

Comment:
any particular reason for using named return values?

Reply:
no. Will it cause an issue?

Reply:
no functional issues, but format is inconsistent with code base. I think it's a bit less readable

Reply:
I believe It is more clear about what is returned and We don't need to initialize return variables.

Reply:
@bitscuit Would you mind reviewing it again? cc @Daniel-Fan 
I added the same function for the operandconfig as well.

Reply:
I would believe it is acceptable to use return value since there are multiple `return` in the function and the return value is not simply `bool` or `error`.

Reply:
the function is probably short enough that this is okay. We should be careful with using named return values though. It can be hard to figure out what is being returned since the return statement could be far away from where the variable is assigned a value